typescript/deploy/src/deploy.ts

```diff
@@ -107,6 +107,9 @@ export abstract class AbacusDeployer<
       implementation.address,
       ubcAddress,
     );
+    // Wait for the beacon to be deployed so that the proxy
+    // constructor is happy.
+    await beacon.deployTransaction.wait(chainConnection.confirmations);
```

Comment:
🙏
we were considering adding these `wait` at the provider level but not sure it's worth departing from the ethers interface